app/assets/javascripts/spree/frontend/add_recently_viewed_products.js.coffee

```diff
@@ -0,0 +1,5 @@
+Spree.fetch_recently_viewed_products = (productId = "")->
+  $.ajax
```

Comment:
@damianlegawiec I think we should only send request if `productID` is not blank.